Modules/Core/Common/test/itkNumericTraitsTest.cxx

```diff
@@ -1298,5 +1298,21 @@ itkNumericTraitsTest(int, char *[])
   // CompileTime Checks IsComplex traits does not return
   CheckIsComplexTraits();
 
+  itkConceptMacro(NumericTraitsCheck1, (itk::Concept::HasNumericTraits<long>));
+  itkConceptMacro(NumericTraitsCheck2, (itk::Concept::HasNumericTraits<std::complex<float>>));
+  itkConceptMacro(NumericTraitsCheck3, (itk::Concept::HasNumericTraits<itk::Vector<float, 3>>));
+  itkConceptMacro(NumericTraitsCheck4, (itk::Concept::HasNumericTraits<std::vector<float>>));
+  // itkConceptMacro(NumericTraitsCheck5, (itk::Concept::HasNumericTraits<itk::Array<long double>>));
+  itkConceptMacro(NumericTraitsCheck6, (itk::Concept::HasNumericTraits<itk::VariableLengthVector<double>>));
+  // itkConceptMacro(NumericTraitsCheck8, (itk::Concept::HasNumericTraits<itk::FixedArray<double, 3>>));
+  // itkConceptMacro(NumericTraitsCheck10, (itk::Concept::HasNumericTraits<itk::Matrix<double, 3, 3>>));
+  // itkConceptMacro(NumericTraitsCheck11, (itk::Concept::HasNumericTraits<itk::CovariantVector<double, 3>>));
+  // itkConceptMacro(NumericTraitsCheck12, (itk::Concept::HasNumericTraits<itk::SymmetricSecondRankTensor<double, 3>>));
+  // itkConceptMacro(NumericTraitsCheck13, (itk::Concept::HasNumericTraits<itk::DiffusionTensor3D<double>>));
```

Comment:
Why are these lines commented out?

Reply:
They failed the concept check due to atleast a missing IsPositive trait.

Reply:
OK, thanks Bradley. Do you think it's important that those concept checks should also be fixed? If so, could you maybe add a comment to the source code why these concept check are still commented out? If it's important enough, you could also make it an "issue", of course  😃  

On the other hand, if there is no need to make those commented-out checks work, you might as well remove them 😺

Reply:
Honestly, for this particular pull request I think it would be fine to just check `itk::Vector`, as that's what the PR is about in the first place.

By the way, did you notice a typo ("HasNumircTraits") in the subject line, and the commit message? No showstopper to me, of course.

Reply:
Yes. I manually wrote a couple of these cases and then Github Copilot wrote the rest. It does a reasonable job of generating some patterned test cases.

I was surprised that the commented out pixel types failed the concept. But I don't think this is a regression, those pixel types just never passed it.

I cleaned this up and re-pushed.